kolibri/core/assets/src/styles/core-theme.styl

```diff
@@ -29,50 +29,9 @@ $core-text-alert-bg = #FDFADF
 $radius = 4px
 $core-time = 0.25s
 
-
-/*
-  Styles below were moved from a Learn-specific style-sheet to here.
-  It was difficult to separate them because the $portrait-breakpoint
-  is defined in terms of the card width.
-
-  TODO: this needs to be cleaned up and organized significantly.
-  At the very least these variables need to be namespaced.
-*/
-
-$thumbnail-height = 122px     // fixed thumbnail height
-
-$card-height = 200px         // fixed card height
-$card-width = 200px         // fixed card width
-$card-gutter = 20px         // spacing between columns
-$n-cols-array = 1 2 3 4 5 6    // possible numbers of columns
-
 $nav-width = 75px
 $left-margin = 100px
 $right-margin = 25px
 
-$horizontal-card-width = 320px
-$horizontal-card-height = 100px
-
-$learn-toolbar-height = 42px
-
-// compute the target grid width based on the number of columns
-grid-width($n-cols)
-  ($card-width * $n-cols) + ($card-gutter * ($n-cols))
-
-$portrait-breakpoint = $left-margin * 2 + grid-width(1) + $right-margin
+$portrait-breakpoint = 520px // TODO - compute this based on variables
```

Comment:
It used to be calculated using card widths... which was a problem. Hardcoded now, but we can discuss better methods later.